proxy/ReverseProxy.cc

```diff
@@ -136,7 +136,13 @@ reloadUrlRewrite()
     // Hold at least one lease, until we reload the configuration
     newTable->acquire();
 
-    ink_atomic_swap(&rewrite_table, newTable)->release(); // Swap configurations, and release the old one
+    // Swap configurations
+    oldTable = ink_atomic_swap(&rewrite_table, newTable);
```

Comment:
In what case is oldTable == nullptr?

Reply:
AIUI, that happens when users run `traffic_server -C verify_config` (#4466). In that case `reloadUrlRewrite()` is called by `cmd_verify()`.

Reply:
Hmmm, does it seem right that we'd swap the configuration if it's a "verify_config" event? I'll have to check how all that works, I'm not faimiliar with how the verify_config was expected to work in the past.

Reply:
@zwoop: you're right, that should probably not be the case! I've opened https://github.com/apache/trafficserver/pull/4624 to address the issue.